sncast_scripts/Scarb.toml

```
@@ -0,0 +1,25 @@
+[package]
+name = "sncast_scripts"
+version = "0.17.0"
```

```suggestion
version = "0.1.0"
```

Does this need to be pegged to the latest release from the library?

---

Mmm no, probably not. Good call